daemon/api_test.go

```diff
@@ -6852,7 +7042,8 @@ func (e fakeNetError) Error() string   { return e.message }
 func (e fakeNetError) Timeout() bool   { return e.timeout }
 func (e fakeNetError) Temporary() bool { return e.temporary }
 
-func (s *appSuite) TestErrToResponseNoSnapsDoesNotPanic(c *check.C) {
+func (s *apiSuite) TestErrToResponseNoSnapsDoesNotPanic(c *check.C) {
+	c.Skip("silly one")
```

Comment:
drop the test, or the skip :-)

Reply:
oops, thanks for spotting this, actually I added the skip to make sure the other new one was covering things, but I think we still need this one (it's about snaps being empty). Removing the skip